da-indexer/da-indexer-logic/src/eigenda/proto/disperser/disperser.proto

```
@@ -1,267 +1,271 @@
 syntax = "proto3";
 package disperser;
+
 import "common/common.proto";
```

Buf (1.54.0)

import "common/common.proto": file does not exist

(COMPILE)